code/datums/gamemode/objectives/ninja/stealsake.dm

```diff
@@ -0,0 +1,13 @@
+/datum/objective/stealsake
+    name = "\[Ninja\] Steal a bottle of sake"
+    explanation_text = "You have gone without a drink for far too long. Steal a bottle of sake, the only beverage worthy of your consumption."
```

Comment:
Avoid fluffy explanation text for objectives, since this is what gets displayed on the scoreboard with SUCCESS or FAIL after it. This is the same reason we don't have vampires jecties written like, "We must taste the sweet nectar of blood. Consume 240u of crewman blood, and be wary, only the worthy blood of souled individuals will pass your high standards."

Note that freeform jecties are explicitly the exception to this rule.

Reply:
What about `"You have gone without a drink for far too long. Steal a bottle of Uchuujin Junmai Ginjo Sake."`? It's shorter than the dominance over AIs jectie (the original one was 15 characters longer than the AI objective).

Reply:
Alright, although I should say that the dominance objectives shouldn't be a standard because I was really pushing the limits with those as it was.

Reply:
Alright, let's cut out all the ARRPEE then.